homepage/templates/homepage/reviews/reviews.html

```diff
@@ -1,18 +1,77 @@
 </head>
-
 <body>
-{% extends 'homepage/app_layout.html' %}
-{% block content %}
-<div class="container" style="margin-top:30px">
-  <div class="row">
-    <div class="col-sm-4">
       <h2>Reviews Page</h2>
+      <div class="num-review-title">{{reviews.count}} reviews available</div>
+      <div class="page-container" style="margin-top:30px">
+        <div class="reviews-container">
+            {% for review in reviews %}
+            <div class="review-box">
+             <div class="title-row">
+              <h4>{{review.user}}</h4>
+              <h6>{{review.date}}</h6>
+        </div>
+        <div>
+              About Course: {{review.course.name}}
+        </div>
+        <div style="margin-top: 20px">
+              {% if review.rate == 5 %}
+              {% for i in "12345"|make_list %}
+              <img src="{% static 'icons/star.png' %}">
+              {% endfor %}
+              {% elif review.rate == 4 %}
+              {% for i in "1234"|make_list %}
+              <img src="{% static 'icons/star.png' %}">
+              {% endfor %}
+              {% elif review.rate == 3 %}
+              {% for i in "123"|make_list %}
+              <img src="{% static 'icons/star.png' %}">
+              {% endfor %}
+              {% elif review.rate == 2 %}
+              {% for i in "12"|make_list %}
+              <img src="{% static 'icons/star.png' %}">
+              {% endfor %}
+              {% else %}
+              <img src="{% static 'icons/star.png' %}">
+              {% endif %}
+        </div>
+        <p style="margin-top: 20px">
+              {{review.content}}
+              <div class="extra-details">
+               {% if review.professor %}
+               <div>
+                     <label class="detail-label">Taught by: </label>{{review.professor}}
+               </div>
+               {% endif %}
+               <div>
+                <label class="detail-label">Course load: </label>
+                {% if review.course_load == 5 %}
+                <img class="load-emoji" src="{% static 'icons/smiling.png' %}">
+                {% elif review.course_load == 4 %}
+                <img class="load-emoji" src="{% static 'icons/smile.png' %}">
+                {% elif review.course_load == 3 %}
+                <img class="load-emoji" src="{% static 'icons/neutral.png' %}">
+                {% elif review.course_load == 2 %}
+                <img class="load-emoji" src="{% static 'icons/sad.png' %}">
+                {% else %}
+                <img class="load-emoji" src="{% static 'icons/crying.png' %}">
+                {% endif %}
+          </div>
     </div>
-  </div>
+    <div class="row likes-row">
+         <img class="like-btn" src="{% static 'icons/thumb-up.png' %}"> {{review.likes_num}} students found this helpful
+   </div>
+</p>
 </div>
-<p class="EOM">(End of messages)</p>
-{% endblock %}
+{% endfor %}
+</div>
+</div>

```

Comment:
these tags for example lose their indentation 
please try to reformat this page to get it better indentation
In PyCharm is: CTR+ALT+L